samples/net/echo_server/src/tcp.c

```diff
@@ -6,11 +6,8 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
-#if 1
-#define SYS_LOG_DOMAIN "echo-server"
-#define NET_SYS_LOG_LEVEL SYS_LOG_LEVEL_DEBUG
-#define NET_LOG_ENABLED 1
-#endif
+#define LOG_MODULE_NAME echo_server_tcp
+#define NET_LOG_LEVEL LOG_LEVEL_DBG
```

Comment:
Almost all of the samples are using SYS_LOG_LEVEL_DEBUG elsewhere in the PR.  Should we be using LOG_LEVEL_DBG (like here) instead?